### PR TITLE
Replace Zulip with Discord in README

### PR DESCRIPTION
**Pull request by x37v**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ The development of iced is sponsored by the [Cryptowatch] team at [Kraken.com]
 [The Elm Architecture]: https://guide.elm-lang.org/architecture/
 [the current issues]: https://github.com/hecrj/iced/issues
 [contributing guidelines]: https://github.com/hecrj/iced/blob/master/CONTRIBUTING.md
-[Zulip server]: https://iced.zulipchat.com/
+Iced [Discord server]: https://discord.gg/3xZJ65GAhd
```
Comment (hecrj) [inline README.md:217]: The `Iced` word here is out of place. It should be:

```suggestion
[Discord server]: https://discord.gg/3xZJ65GAhd
```

**Review by hecrj — CHANGES_REQUESTED** [review]
Yes, good catch!

Just a small thing we need to fix.

**Review by hecrj — APPROVED** [review]
Thank you! :tada:

**Comment by x37v** [comment]
> Yes, good catch!
> 
> Just a small thing we need to fix.

yeah.. I added `Iced` specifically do differentiate between it and the rust sever below it but, if it makes sense to just call it "Discord Server" .. that is fine by me

**Pull request merged**